index.html

```diff
@@ -0,0 +1,187 @@
+<!doctype html>
+<html lang="en">
```

Comment:
❗️ 수정요청
html lang 속성은 스크린리더, 검색 엔진 등에게 해당 사이트가 어떤 언어 기반으로 작성되었는지 설명하는 속성입니다.
해당 속성에는 특정 언어들을 의미하는 예약어가 들어가야 합니다. 예를들어 en은 english 즉 영어를 의미합니다.
저희 사이트는 한국어 기반이므로 ko를 써야합니다~

```suggestion
<html lang="ko">
```